service/arm_templates/security_groups_setup.json

```diff
@@ -54,7 +57,7 @@
               "sourcePortRange": "*",
               "destinationPortRange": "*",
               "sourceAddressPrefix": "*",
-              "destinationAddressPrefix": "[parameters('masterSubnetCidr')]",
+              "destinationAddressPrefix": "*",
```

Comment:
We should deny all traffic, not only one for our subnet (i.e. it does not deny pods traffic).